atm.js

```diff
@@ -1,21 +1,45 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checkingWithdrawButton = $("#withdrawButton");
+var checkingBalance = $("#checkingBalance");
+var checkingNewDeposit = $("#valueinput");
+var checkingValue = $("#checkingNewDeposit.val()");
+var cBalance= checkingBalance.html().substr(1);
+var checkingDespositButton = $("#depositButton");
+checkingDespositButton.on("click", function(){
+  var checkingNewDeposit = $("#valueinput");
+  var checkingTotal = parseInt(checkingValue) + parseInt(cBalance);
+  var cBalance= checkingBalance.html().substr(1);
+    console.log(checkingTotal)
+    checkingBalance.html("$" + checkingTotal)
+})
+checkingWithdrawButton.on("click", function(){
+  var checkingTotal = parseInt(checkingValue) - parseInt(cBalance);
+    console.log(checkingTotal)
+    checkingBalance.html("$" + total)
+})
```

Comment:
Excellent Job on this assignment Christine!!! :star2: The code below looks very similar to the code above. Can you think of a way to handle both accounts without two separate functions and click handlers? i.e. what if you wanted the user to be able to create another checking or savings account?